src/server/ocpp/services/OCPPService.ts

```diff
@@ -371,8 +374,14 @@ export default class OCPPService {
             transaction.numberOfMeterValues >= 1) {
             transaction.phasesUsed = Utils.getUsedPhasesInTransactionInProgress(chargingStation, transaction);
           }
-          // Handle OCPI
-          await OCPPUtils.processOCPITransaction(headers.tenantID, transaction, chargingStation, TransactionAction.UPDATE);
+          // Indicator needed to decide if transaction is handled with either Gireve or Hubject.
+          if (transaction.ocpiData?.session) {
```

Comment:
Interesting, that way both roaming system can be enable on one tenant ...

Reply:
Yes, I think I should be possible to activate multiple platforms with a few changes in the code.
Only for the authorization process there should be a rule:
For example: 
For an unknown roaming user 
1. try authorization with Gireve 
   -> if authorized -> start OCPI session
   -> if not authorized 2.  try authorization with Hubject
        -> if authorized -> start OICP session
       -> if not authorized -> reject user

in the following processes the if conditions 'if (transaction.ocpiData?.session)'  and 'if (transaction.oicpData?.session)' should ensure that always the right platform is used 

a more advanced concept should give the CPO the option to prioritize one eRoaming platform over the other.

Reply:
okay, I'll do that